### ISSUE TITLE
is this project beta? who owns it?

### ISSUE THREAD

**Issue by cs01** [kennethreitz/pipenv#1089]
I think Pipenv is great, but am confused on the state of things. A single, comprehensive, canonical source of information on the topic would be extremely helpful (preferably not under someone's personal github account).

This git repo (Pipenv) claims it is the official packaging tool for Python
> Pipenv — the officially recommended Python packaging tool from Python.org, free (as in freedom).

Yet it's stored under the @kennethreitz git user. This seems odd. Why not under the Python Packaging Authority github user @pypa or Python @python?

The [Python Packaging Authority](https://github.com/pypa/pipfile) has this to say about Pipfile and Pipfile.lock
>  WARNING: This project is under active design and development. Nothing is set in stone at this point of time.
This repository contains the design specification of the Pipfile format, as well as (soon) an implementation of a parser for the specification which can be used by pip and any other consumer (e.g. Pipenv), once the design (including the form of a Pipfile itself) has been built out and finalized.

So it seems a surprising that the officially recommended tool is "under active design an development" and that "nothing is set in stone at this time". Is this still accurate? Is it intentional that Pipenv be distributed to thousands or millions of machines if it's still in beta?

**Comment by techalchemy** [comment]
@cs01 first of all, I am confused as to why you take issue with a project being under someone's github account.  @kennethreitz created the project, he is the owner and that's just what the situation is, I'm not sure why that is difficult to believe.

> This git repo (Pipenv) claims it is the official packaging tool for Python
>>  Pipenv — the officially recommended Python packaging tool from Python.org, free (as in freedom).

You say that we _claim_ this, which comes across kind of hostile.  Are you calling it into question?  I'm not really following the point of this. The repository is what it is, you have already read what the PyPA says about it.  Whether it will stay under Kenneth's personal account for eternity, that is another question.  For now, it resides here. If you want to open an issue on that particular point in a more productive manner I'm sure that's a discussion we can have.

> So it seems a surprising that the officially recommended tool is "under active design an development" and that "nothing is set in stone at this time". Is this still accurate? 

Yes. Nearly every piece of software you use is "under active design and development".  Pip is about to release version 10 which will change core functionality in numerous ways.  Are you opening issues there asking them to stop development? Should we put away the keyboards and stop improving things?

> Is it intentional that Pipenv be distributed to thousands or millions of machines if it's still in beta?

[Here is a link to the semantic versioning spec.](http://semver.org/) We are not in beta. Actively developing something does not make it a beta release. The version of the software determines if it is beta or not.

I am closing and locking this because I don't think your goal was to have a productive conversation.

**Comment by cs01** [comment]
[copying my comment from #1080]

And FWIW I was not trying to be snarky or hostile in #1089. I am genuinely interested in understanding how this tool fits in with the other tools.

pip and virtualenv are both owned by the @pypa user, so it surprised me that an official tool like pipenv doesn't live there. I take issue because it concerns me that an officially recommended tool is controlled by an individual and not the community. Now it's entirely possible the community decides what gets merged in this repo and that Kenneth doesn't have the last say. But I don't see any documentation on how decisions and changes are made to pipenv, so I asked the question. At the very least it's inconsistent to have some of the tools in pypa and some in kennethreitz.

I am only asking these questions because of the massive importance and influence a tool like pipenv has in the programming world. Hundreds of thousands/millions of developers rely or will rely on it.

pypa/pip claims it is "The PyPA recommended tool for installing Python packages" while pipenv says it's "the officially recommended Python packaging tool from Python.org". These are very similar, but not identical I guess. In any case it confuses me.

My interpretation of pypa/Pipfile was that Pipfile is basically not ready for production use because the specification is not entirely settled on. If that's not true, I am happy to be corrected. Again, I'm not trying to criticize, just trying to understand. I totally get that software is always in development, but that is different from deploying software based on a specification that warns you that it's unfinished. And if that's the case, that's fine, but maybe it should be noted that deploying Pipfiles to production is not recommended until the specification is settled on, etc.

In summary, I'm not trying to just trash talk or something, I'm truly trying to understand how all the pieces fit together. I would like to continue the conversation in that ticket if possible. Thanks for taking the time to respond, and thanks for helping to maintain pipenv.

**Comment by erinxocon** [comment]
@kennethreitz can chime in on the status of the project.  He is involved with the PSF and on the board.

**Comment by techalchemy** [comment]
^ That, but roughly speaking all of the tools you mentioned are maintained by a team of core maintainers.  I think there is merit to discussing splitting an organization for this, we've talked about it in the past but we didn't think the project was big enough to merit it yet.  It might be time to revisit.

**Comment by kennethreitz** [comment]
no

**Comment by kennethreitz** [comment]
@cs01 sorry for the aggressiveness present here -- it is being dealt with internally.

Basically, we are *not* an official PyPA project, simply a tool that they are (or will soon be) recommending users use. There's a big difference between those two things. In the future, this may or may not change, it does not matter today, however. 

Hope that helps clarify things. Sorry again for the tone of your previous interactions with this project's representatives, which are not considered either acceptable nor representative of this project's values or goals for the community. 

Much love,
Kenneth Reitz
✨🍰✨

**Comment by douglasmiranda** [comment]
This issue may need an official comment on the subject: https://github.com/docker-library/python/issues/258

**Comment by ncoghlan** [comment]
Just noting for the record that `pipenv` *did* subsequently migrate under the PyPA GitHub org: [https://github.com/pypa/pipenv/](https://github.com/pypa/pipenv/)

I thought we had an issue for that transfer, but I can't find it now, so it must have happened live while Kenneth and I were discussing the possibility.
